__tests__/unit/old-structure/tree-fetching/stratetegies/cache-with-fallback-to-network.spec.ts

```diff
@@ -67,9 +66,8 @@ describe('Utils: tree fetching (load: cache-with-fallback-to-network)', () => {
       strategy,
       retry,
     })).rejects.toEqual([
-      new BeagleCacheError(url),
       // @ts-ignore
-      new BeagleNetworkError(url),
+      new BeagleNetworkError(url,undefined,500,'GET'),
```

Comment:
```suggestion
      new BeagleNetworkError(url, undefined, 500, 'GET'),
```
It would be nice to add the space after the "," to keep a pattern.